avm/res/sql/instance-pool/tests/e2e/defaults/main.test.bicep

```
@@ -19,7 +19,7 @@ param namePrefix string = '#_namePrefix_#'
 
 // Note, we set the location of the NetworkWatcherRG to avoid conflicts with the already existing NetworkWatcherRG
 #disable-next-line no-hardcoded-location
-var enforcedLocation = 'uksouth'
+var enforcedLocation = 'northeurope'
```

```suggestion
var enforcedLocation = 'uksouth'
```
Would need to remain `ukSouth` in upstream as that's where the persistent resource is deployed.